okhttp/src/commonMain/kotlin/okhttp3/TlsVersion.kt

```diff
@@ -15,6 +15,9 @@
  */
 package okhttp3
```

Comment:
What is the pattern for 

```

  @JvmName("-deprecated_javaName")
  @Deprecated(
      message = "moved to val",
      replaceWith = ReplaceWith(expression = "javaName"),
      level = DeprecationLevel.ERROR)
  fun javaName(): String = javaName
```

How to only implement in JVM and maintain absolute backwards compatibility?

Reply:
We need to create 3 copies of the class:
 * commonMain defines the shared API with expect functions, plus an internal file with shared implementations
 * jvmMain defines actuals, delegating to the shared implementations. Plus JVM-only APIs
 * nonJvmMain is the same as jvmMain, but no JVM-only APIs

Kotlin multiplatform needs a lot of boilerplate if there’s any API differences

Reply:
I got this working, which was really helpful.  Then realised we probably don't want to bring across javaName to other platforms.